ide-common/src/main/kotlin/org/digma/intellij/plugin/codelens/AbstractCodeLensService.kt

```diff
@@ -158,7 +157,6 @@ abstract class AbstractCodeLensService(private val project: Project): Disposable
         FileEditorManager.getInstance(project).allEditors.forEach {
             if (it.file == psiFile.virtualFile && it is TextEditor) {
                 Log.log(logger::debug, "restarting DaemonCodeAnalyzer for {}", psiFile.virtualFile)
-                CodeVisionPassFactory.clearModificationStamp(it.editor)
```

Comment:
what happened to it? don't we need it?

Reply:
the class changed to internal and can't be used. it works without it